### PR TITLE
S001 add auditing checks

### PR DESCRIPTION
**Pull request by miagreen23**

### CODE REVIEW

File: dependency-reduced-pom.xml
Comment (glenn-hamilton) [inline dependency-reduced-pom.xml:1]: This file shouldn't be here

File: .github/workflows/maven.yml
Comment (glenn-hamilton) [inline .github/workflows/maven.yml:1]: This file is empty

File: dependency-reduced-pom.xml
Comment (miagreen23) [reply dependency-reduced-pom.xml:1]: Removed

File: .github/workflows/maven.yml
Comment (miagreen23) [reply .github/workflows/maven.yml:1]: So it is

Comment (glenn-hamilton) [reply .github/workflows/maven.yml:1]: Or is it...

**Review by glenn-hamilton — APPROVED** [review]

File: .github/workflows/maven.yml
Comment (miagreen23) [reply .github/workflows/maven.yml:1]: I don't know anything anymore </3

**Review by siyandlovu1 — APPROVED** [review]

**Pull request merged**